readers/postgres/messages_test.go

```diff
@@ -38,13 +38,13 @@ var (
 func TestReadSenml(t *testing.T) {
 	writer := pwriter.New(db)
 
-	chanID, err := uuidProvider.New().ID()
+	chanID, err := uuid.New().ID()
```

Comment:
There is no need to instantiate the ID provider for each ID. You can create one provider and use it for all IDs in this test. Check all the other places in tests where you have multiple instances.